src/App.js

```diff
@@ -31,6 +32,12 @@ function App() {
         <Route path="/checkout/completed" element={<CheckoutComplete />} />
         <Route path="/profile" element={<UserProfile />} />
         <Route path="*" element={<PageNotFound />} />
+        <Route
+          path="/chat"
+          element={
+            <AlbumChat apiUrl="https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy" />
```

Comment:
In fact, maybe it's even better to use `AlbumPreview` as the component you're testing, just to make sure that the chat component you've nested in there looks good / works well altogether 

```suggestion
            <AlbumPreview apiUrl="https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy" />
```